modules/bitgo/test/v2/unit/internal/tssUtils/ecdsa.ts

```diff
@@ -664,7 +679,7 @@ describe('TSS Ecdsa Utils:', async function () {
         to: SignatureShareType.BITGO,
         share: userSShare.R + userSShare.s + userSShare.y + userOmicronAndDeltaShare.dShare.delta + userOmicronAndDeltaShare.dShare.Gamma,
       };
-      const getBitGoSShare = MPC.sign(signablePayload, getBitgoOShareAndDShares.oShare, userOmicronAndDeltaShare.dShare);
+      const getBitGoSShare = MPC.sign(signablePayload, getBitgoOShareAndDShares.oShare, userOmicronAndDeltaShare.dShare, createKeccakHash('keccak256') as Hash);
```

Comment:
Previously, this `beforeEach()` was a bit bugged. `createUserSignShare()` correctly uses a keccack hash, but calling `MPC.sign()` with no hash specified makes it default to sha256 which means the signatures produced were invalid. 

Figure we might as well generate actually useful signatures here as another bit of sanity checking/regression testing